<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resumeButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testFireAlarmButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TestFireAlarm</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="E1_Floor3">
       <property name="text">
        <string>E1_Floor 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="E1_Floor5">
       <property name="text">
        <string>E1_Floor 5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Floor3">
       <property name="text">
        <string>Floor 3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="E2_Floor3">
       <property name="text">
        <string>E2_Floor 3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="E2_Floor5">
       <property name="text">
        <string>E2_Floor 5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Floor2">
       <property name="text">
        <string>Floor 2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="E2_Floor2">
       <property name="text">
        <string>E2_Floor 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="Elevator2">
       <property name="text">
        <string>Elevator2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="E2_Floor1">
       <property name="text">
        <string>E2_Floor 1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="E1_Floor4">
       <property name="text">
        <string>E1_Floor 4</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="E1_Floor1">
       <property name="text">
        <string>E1_Floor 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Elevator1">
       <property name="text">
        <string>Elevator1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Floor5">
       <property name="text">
        <string>Floor 5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Floor1">
       <property name="text">
        <string>Floor 1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="E1_Floor2">
       <property name="text">
        <string>E1_Floor 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Floor4">
       <property name="text">
        <string>Floor 4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="E2_Floor4">
       <property name="text">
        <string>E2_Floor 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="logConsole">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>231</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="floor5Down">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 5 Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor4Down">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 4 Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor3Down">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 3 Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor2Down">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 2 Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor4Up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 4 Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor3Up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 3 Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor2Up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 2 Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor1Up">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor 1 Up</string>
    </property>
   </widget>
   <widget class="QComboBox" name="passengerSelector">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Passenger 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Passenger 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Passenger 3</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="elev1Buttons">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator 1</string>
    </property>
    <widget class="QPushButton" name="elev1Floor5Btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev1Floor4Btn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev1Floor3Btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev1Floor2Btn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev1Floor1Btn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev1OpenBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev1CloseBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev1HelpBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>elev1HelpBtn</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="elev2buttons">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator 2</string>
    </property>
    <widget class="QPushButton" name="elev2Floor5Btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev2Floor4Btn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev2Floor3Btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev2Floor2Btn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev2Floor1Btn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev2OpenBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev2CloseBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="elev2HelpBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>elev2HelpBtn</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="handleHelpButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Handle Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="emergencyStatusLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergency Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overloadElev2Btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulate Overload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reduceLoadElev2Btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reduce Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reduceLoadElev1Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reduce Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="overloadElev1Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulate Overload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerOutButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Out!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
